src/DSManager.cpp

```diff
@@ -367,14 +381,14 @@ Participant* DSManager::removeParticipant(
 }
 
 void DSManager::addSubscriber(
```

Comment:
```suggestion
void DSManager::addDataReader(
```

I suggest applying this renaming to all the other methods, as well as renaming Publisher for DataWriter.